course-v2/layouts/partials/resources_header.html

```diff
@@ -22,6 +22,11 @@ <h2>Download</h2>
       </div>
     </div>
   </div>
+  <div class="alert-warning p-2 touch-device-only">
```

Comment:
alert-warning is an existing class and style that we have in our code. I think it is used in one other place related to videos.

It's a generic class for our warning messages.

Reply:
I meant the`.touch-device-only` class

Reply:
`display: none` is the default rule. The relevant code is under the media query which has `display: block`.

I understand your perspective. I look at this from an engineer's perspective. When you see a code like `<div class="touch-device-only">` it gives the idea that this element will only be rendered for touch devices. Which is the behavior of this class.

And since this class can be used in other places and is not limited to warning messages, I thought a more generic name would be more suitable.

Let me know if you still find the name ambiguous.

PS: I switched it to `limited-pointer-only` because we now use it in places of limited pointer availability.